File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> newNums(nums.size() * 2);
        for(int i = 0; i < nums.size(); i++){
            newNums[i] = nums[i];
            newNums[nums.size() + i] = nums[i];
        }
        return newNums;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = dict()
        for k, v in enumerate(nums):
            diff = target - v
            if v in hashMap:
                return [hashMap[v], k]
            hashMap[diff] = k
        return []